### DISCUSSION TITLE
World entity support for :box function

### DISCUSSION THREAD

**Discussion by Isemenuk27** [wiremod/wire#2793] [Suggestions]
Getting maps bounds would be very useful.

add world support for:
  entity:boxMax()
  entity:boxMin()
  entity:boxSize()
  entity:boxCenter()

**Comment by Isemenuk27** [comment]
local MinS, MaxS = game.GetWorld():GetInternalVariable( "m_WorldMins" ), game.GetWorld():GetInternalVariable( "m_WorldMaxs" )

Looks like MaxS height is incorrect, but X and Y are ok.

Also tried to get this values with .bsp parser, and they`re same.

**Comment by vurvdev** [comment]
Relaying my message from discord:

Can you really represent a map as a cube? Also would be better as separate functions rather than adding overhead to the box functions

**Reply by Isemenuk27** [reply]
It represents map as Rectangle ( +far top and -far bottom points ) and doesn't include 3d skybox. 
You right, maybe it would be better to make separate functions: worldMax(), worldMin(), worldSize(), worldCenter(), worldHeight() to make it less confusing and save invalid entity output ( vec(0) ).
